File: Beecrowd/Beginner/1101 - 1190/1158 - Sum of Consecutive Odd Numbers III/C/1158_SumofConsecutiveOddNumbersIII.c
#include <stdio.h>

int main() {
    int number, x, y;
    scanf("%d", &number);

    for (int i = 0; i < number; i++) {
        scanf("%d %d", &x, &y);

        if (x % 2 == 0) x++;

        int sum = 0;

        for (int j = 0; j < y; j++) {
            sum += x;
            x += 2;
        }

        printf("%d\n", sum);
    }

    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1158 - Sum of Consecutive Odd Numbers III/Python/1158_SumofConsecutiveOddNumbersIII.py
number = int(input())

for _ in range(number):
    x, y = map(int, input().split())

    if x % 2 == 0:
        x += 1

    sum = 0

    for j in range(y):
        sum += x
        x += 2

    print(sum)
